src/FakeItEasy.Analyzer/Resources.resx

```diff
@@ -117,6 +117,19 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="NonVirtualSetupDescription" xml:space="preserve">
+    <value>Non virtual members can not be intercepted.</value>
+    <comment>An optional longer localizable description of the diagnostic.</comment>
+  </data>
+  <data name="NonVirtualSetupMessageFormat" xml:space="preserve">
+    <value>Non virtual member '{0}' cannot be intercepted.</value>
+    <comment>The format-able message the diagnostic displays.</comment>
+  </data>
+  <data name="NonVirtualSetupSpecificationTitle" xml:space="preserve">

```

Comment:
This name is incorrect. The diagnostic doesn't contain "Specification", so we don't see the name in the Solution Explorer:

![image](https://cloud.githubusercontent.com/assets/3275797/18183002/516a496a-7060-11e6-9977-36406c71d5f3.png)

The fix is to rename to `NonVirtualSetupTitle`.

(kudos to @thomaslevesque for noticing the problem)